### ISSUE TITLE
Homing on corexy machines

### ISSUE THREAD

**Issue by Faisalbelwadi** [robottini/grbl-servo#12]
I have made a drawing machine which is a corexy machine wih 2 axis and servo support
Having grbl 0.9i 

I have setup limit switches and am pretty sure the hardware is perfect as hard limits are triggered on both x and y successfully

BUT when i run a homing cycle, both x and y start to move good, but then when one of the switches is triggered, the second axis doesn't move (or moves vert very slow) and does not home and makes a strong humming sound and homing fails

Note: if i deliberately press the switch while humming, there is no homing fail.
In short only one axis gets homed properly

Please can you provide me help with this matter as I've searched all over the internet but nothing worked
